src/parsing/css-style-value-parsing.js

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//     You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(internal) {
+
+  var parsers = {};
+  parsers[CSSNumberValue] = internal.parsing.consumeNumber;
+  parsers[CSSLengthValue] = internal.parsing.consumeLength;
+  parsers[CSSTransformValue] = internal.parsing.consumeTransform;
+
+  function consume(consumer, string) {
```

Comment:
This is a helper function to deal with the fact that there may not be an entry in the parsers map above, and avoid writing if (!function) everywhere. Otherwise below you have to write (which isn't really that bad I suppose). Anyway something to think about.

```javascript
for (var i = 0; i < supportedStyleValues.length; i++) {
  var consumer = parsers[supportedStyleValues[i]];
  var parsed = null;
  if (consumer) {
    parsed = consumer(property, string);
  }
  if (parsed) {
  return parsed;
}
```

If we keep it, should probably give this a more descriptive name, could be "maybeConsume"?